---
content_type: page
description: This section includes brief descriptions of topics covered in each class
  session.
hide_download: true
hide_download_original: null
learning_resource_types:
- Lecture Notes
ocw_type: CourseSection
title: Lecture Summaries
uid: 944b9a5e-db4a-4be7-c81e-4e7b400577be
---

{{< tableopen >}}
{{< theadopen >}}
{{< tropen >}}
{{< thopen >}}
week #
{{< thclose >}}
{{< thopen >}}
topics
{{< thclose >}}
{{< thopen >}}
lecture summaries
{{< thclose >}}

{{< trclose >}}

{{< theadclose >}}
{{< tropen >}}
{{< tdopen >}}
1
{{< tdclose >}}
{{< tdopen >}}
Introduction to the Course
{{< tdclose >}}
{{< tdopen >}}


{{< anchor "L1summary" >}}{{< /anchor >}}During the first class, the instructors and students will introduce themselves. The instructors will summarize the course topic and describe their connection to the field, review the syllabus, schedule a class meeting day/time that works for everyone, and discuss how to search the primary literature (PubMed, Google Scholar, NCBI, etc.). This session (and others) will end with a 10–15 minute overview of the topic for next week to provide context for the assigned reading and a brief introduction to unfamiliar experimental methods.


{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
2
{{< tdclose >}}
{{< tdopen >}}
RiPPs (ribosomally synthesized and post-translationally modified peptides): Discovery via Culture/Manipulation of the Native Producer
{{< tdclose >}}
{{< tdopen >}}


{{< anchor "L2summary" >}}{{< /anchor >}}**A classic method to discover natural products is by observing whether a bacterium secretes an active compound.** In 1928, Alexander Fleming famously returned to lab after a two-week vacation and discovered that the _Staphylococcus_ he was working with did not grow near mold on contaminated plates. This led to the discovery that mold secreted penicillin, and thus began the golden age of antibiotics. This technique of screening for a desired activity and then determining what compound(s) generates this activity is still used today, albeit less frequently.

In the paper by Lynch et al., 100 staphylococci strains derived from humans were screened for antimicrobial activity against other bacteria, leading to the discovery of a novel antimicrobial peptide termed “capidermicin.” Stein et al. similarly observed that _Bacillus subtilis_ strain A1/3 generates compounds with a broad range of activities and went on to characterize the two-component lanthipeptide ericin, which was responsible for some of the antimicrobial activity observed from this strain.


{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
3
{{< tdclose >}}
{{< tdopen >}}
RiPPs: Discovery via Genomics/Data-Driven Strategies
{{< tdclose >}}
{{< tdopen >}}


{{< anchor "L3summary" >}}{{< /anchor >}}**With the advent of inexpensive genome sequencing, natural product discovery was revolutionized.** Rather than growing a strain of interest—not always a trivial task—in hopes of finding some activity and then tracking down the component responsible for that activity, the availability of genomic data allows many more efficient approaches. Scientists can search a sequenced genome for a gene product of potential interest, such as one that has no previously characterized homologues or one that based on its genomic context is likely to be subject to novel enzymatic transformations. In fact, one of the major challenges in light of the wealth of genomic data is how one identifies which genes warrant further investigation. Having decided which gene product they want to characterize, they can attempt to obtain it by focusing on and developing methods to culture the native producer and purifying the NP through extraction, affinity chromatography, or other methods.

Alternatively, if the native producer is not tractable to lab work or the gene cluster is not expressed in the native producer, researchers can express the encoding gene(s) in a tractable host, such as _Escherichia coli_, and then isolate the putative peptide natural product via affinity chromatography, usually through affinity to a genetically appended artificial tag. The NP can then be characterized via mass spectrometry, NMR, or activity assays.

In the paper by Zhao et al., the authors used a BLAST search to identify a unique two-component lanthipeptide biosynthetic gene cluster encoded by a _Ruminococcus flavefaciens_, a bacteria resident in the microbiome of cattle. Preliminary analysis suggested that this gene cluster encoded a “two-component” lanthipepide, i.e., one in which two peptides (the “α” and “β” peptides) act together to generate antimicrobial activity. Uniquely, analysis based on homology to known lanthipeptide gene clusters suggested that this cluster encoded an unprecedented 4 α-peptides and 8 β-peptides that together could form 32 different combinatorial two-component lanthipeptides. To test whether this cluster indeed functions as predicted, they then cloned the genes into plasmids and expressed them in _E. coli_. Following expression, they isolated the produced products via immobilized metal ion affinity chromatography (IMAC) and characterized them using matrix-assisted laser desorption ionization time-of-flight mass spectrometry (MALDI-TOF MS).

In the second paper, Schwalen et al. leveraged a “big data” approach, searching for thiopeptide-containing gene clusters using known associated biosynthetic enzymes as a search query, to identify 508 putative thiopeptide-encoding biosynthetic gene clusters. They then chose one cluster containing an uncharacterized biosynthetic gene grouping, grew cultures of the native producer bacterial strain, extracted the cultures with methanol, and purified the putative natural product from the methanolic extracts via several rounds of reversed-phase chromatography. They characterized the resulting product via high resolution mass spectrometry and nuclear magnetic resonance to verify that they had indeed isolated a novel thiopeptide containing a thioamide bond.


{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
4
{{< tdclose >}}
{{< tdopen >}}
RiPPs: Biosynthesis and Enzyme Function/Mechanism
{{< tdclose >}}
{{< tdopen >}}


{{< anchor "L4summary" >}}{{< /anchor >}}**As the RiPP name implies, the post-translational modifications installed by tailoring or modifying enzymes provide the enormous diversity in chemical structures of these NPs (natural products).** Understanding what reactions these enzymes perform and what substrates and cofactors they need is important for gaining insight into NP biosynthetic pathways and knowing how to recapitulate the environment of a native producer within a heterologous host. We will discuss two papers that focus on individual enzymes within the biosynthetic pathways of the highly modified RiPPs NAI-107 and nosiheptide.

Ortega et al. took a closer look at the tRNA substrate required for lanthipeptide dehydrated residue formation in NAI-107. They used substrate mutagenesis and X-ray crystallography studies to assess the modifying enzyme-tRNA interaction and determine elements needed for heterologous host production of this RiPP. The gene cluster for nosiheptide was characterized in 2009, with an initial hypothesis for the order of the biosynthetic steps and the functions of the enzymes. However, this hypothesis has continually evolved over the last decade as more details about the enzymes have been investigated. Based on initial bioinformatic analysis, the biosynthetic enzyme NosN was annotated as a radical S-adenosylmethionine (radical SAM) methylase. However, holes in the biosynthetic pathway remained with no assigned enzyme.

LaMattina et al. delved deeper into the mechanistic details of NosN and proposed that this radical SAM enzyme is not just a methylase but also responsible for the formation of an ester linkage. Their work highlights the extra biochemical experiments needed to test hypotheses from bioinformatic assignments.


{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
5
{{< tdclose >}}
{{< tdopen >}}
RiPPs: Engineering
{{< tdclose >}}
{{< tdopen >}}


{{< anchor "L5summary" >}}{{< /anchor >}}**Since their primary structures are genetically encoded, RiPPs are prime candidates for synthetic biology and engineering pursuits.** Point mutations in the core region of the gene encoding the primary structure of these peptidic NPs are frequently tolerated by the modifying enzymes. Hence, generating variants of these natural projects starts with a simple cloning step and can yield a remarkable variety of products with varying properties.

Ozaki et al. reported the use of an _in vitro_ translation system to generate 52 variants of goadsporin. They then leveraged this information both to better define the order in which the modification enzymes act and to generate designed analogs of goadsporin. Sardar et al. used RiPP enzymatic machinery from cyanobactin synthesis to generate a library of peptide macrocycles.


{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
6
{{< tdclose >}}
{{< tdopen >}}
Panel Discussion with Natural Product Professionals
{{< tdclose >}}
{{< tdopen >}}


{{< anchor "L6summary" >}}{{< /anchor >}}We will assemble a panel of scientists and professionals who have spent time or currently have careers in various fields related to natural product research. We envision assembling a broad panel of careers ranging from biotechnology to patent law, but we also will seek input from the class in the first few weeks about their interests and match the panelists to those interests as much as possible. Topics for conversation include career trajectories, networking strategies, scientific interests, experimental design, and many others.


{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
7
{{< tdclose >}}
{{< tdopen >}}
NRPs (non-ribosomal peptide NPs): Discovery via Genomics/Data-Driven Strategies
{{< tdclose >}}
{{< tdopen >}}


{{< anchor "L7summary" >}}{{< /anchor >}}**The genomic revolution has enabled radical new approaches to discovery of non-ribosomal peptide NPs, including expression of “silent” gene clusters and metabologenomics—studies of the correlation between observed metabolites and the gene clusters that likely produce them.** As we saw with RiPPs, the massive availability of genomic data allows for a wide variety of discovery approaches that were previously not possible, thereby greatly increasing the diversity and number of discovered NRPS products.

Yamanaka et al. used the homologous recombination capabilities of _Saccharomyces cerevisiae_ to assemble a 67-kilobase biosynthetic gene cluster, refactored for a heterologous host, and then express it in _Streptomyces coelicolor_ to yield a novel antibiotic. Goering et al. used an _in silico_ technique to identify likely biosynthetic gene clusters in 178 bacterial strains, grew these native producer strains in four different media, assessed metabolite production via LC-MS, and finally correlated gene clusters to metabolites. This approach allowed them to discover a novel chlorinated nonribosomal peptide.


{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
8
{{< tdclose >}}
{{< tdopen >}}
NRPs: Biosynthesis and Enzyme Function/Mechanism
{{< tdclose >}}
{{< tdopen >}}


{{< anchor "L8summary" >}}{{< /anchor >}}**Examination of the proteins encoded in NRP gene clusters reveals how these NPs are assembled and allows for more nuanced genome mining to find expanded diversity in NP structure.** NRP gene clusters are classically identified by their NRP synthetase (NRPS), an assembly-line biosynthetic enzyme complex. However, just because we can identify these enzymes does not mean we understand how they work. Indeed, establishing the rules of biosynthetic logic and domain interactions within these megaenzymes has been important for applying these enzymes to the production of NRP analogs.

In Kreitler et al., the authors investigated the domain architecture of an NRPS system that includes the production of the bioactive beta-lactone functional group within the final NRP product. They used X-ray crystallography and biochemical assays to propose key domain interactions and examined the substrate scope to determine the capacity for making derivative molecules. Genome mining has also led to the discovery of NRPS-independent pathways for NRP biosynthesis, exemplifying the diversity of enzyme function yet to be uncovered from microbial genomes.

Dunbar et al. used biochemical assays to assign enzyme function within an NRPS-independent NRP pathway. They concluded with a bioinformatics analysis of the beta peptide bond-forming enzyme (an ATP-grasp ligase) to feed back into genome mining studies for the identification of more biosynthetic gene clusters.


{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
9
{{< tdclose >}}
{{< tdopen >}}
NRPs: Engineering
{{< tdclose >}}
{{< tdopen >}}


{{< anchor "L9summary" >}}{{< /anchor >}}**Although NRPS modules cannot be simply swapped in and out as initially hoped, sophisticated engineering strategies have allowed the biosynthetic production of a wide variety of natural product derivatives.** In recent years, advances in cloning and gene assembly have allowed for increasingly sophisticated construction, reconstruction, refactoring, and diversification of NRPS and NRPS-PKS biosynthetic gene clusters. Expression of these constructs has revealed that NRPS/NRPS-PKS modules are not as readily swappable as hoped. However, these studies have also succeeded in diversifying natural products, creating variants with new properties and simultaneously further exploring how the modules interact.

Niquille et al. used a high-throughput assay to develop an NRPS that accepts a β-amino acid (as opposed to an NP α-amino acid). They used this engineered NRPS both to learn how the synthetase can accept the unnatural amino acid and to produce unnatural amino acid–containing nonribosomal peptides. Awakawa et al. used bioinformatic and evolutionary data to guide remodeling of an antimycin-type PKS-NRPS hybrid, allowing creation of unnatural derivatives.


{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
10
{{< tdclose >}}
{{< tdopen >}}
Plant NPs
{{< tdclose >}}
{{< tdopen >}}


{{< anchor "L10summary" >}}{{< /anchor >}}**Plants are prolific generators of NPs.** Our focus will shift for this week from bacteria to plant producers. Genome mining has also played a major role in the study of plant NPs in recent years, but a number of factors make plant NP investigation and engineering very challenging. For one, plants do not have the same gene cluster organization as microbes, making it more difficult to determine the biosynthetic pathway and complete requirements for production of a given NP. Plant NPs can also be challenging to extract in sizeable quantities from the native host and require impractical syntheses, so strategies to produce beneficial compounds in genetically tractable heterologous hosts are highly sought.

In Hodgson et al., the authors aimed to piece together a biosynthetic pathway for limonoids, which have complex chemical structures. The authors used methods such as gene expression analysis and metabolite profiling to fish out candidate genes from the genomes of plant producers. Brown et al. tackled the question of how to produce a compound in a yeast heterologous host once the biosynthesis is known. The authors developed a yeast strain with multiple gene insertions and deletions and demonstrated the production of a precursor to many terpene NPs.


{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
11
{{< tdclose >}}
{{< tdopen >}}
Culture-Dependent and Metagenomic Methods of NP Discovery from Complex Environments
{{< tdclose >}}
{{< tdopen >}}


{{< anchor "L11summary" >}}{{< /anchor >}}**Environments that are home to many microbial species are potentially rich sources for NP discovery.** These environments can include ponds and prairies as well as the human gastrointestinal tract. Some hypotheses to explain the extensive metabolite production from inhabiting microbes stem from the possibility that the microbes might be communicating with the host, communicating with each other, and/or fighting for limited resources.

This week we will delve into two studies that use different methods to classify compounds from the microbiomes of plant leaves and humans. Helfrich et al. performed multiple interaction screens with cultured bacterial strains to classify the types of NPs in the _Arabidopsis thaliana_ leaf microbiome. Cohen et al. took a functional metagenomics approach to investigate gene clusters isolated from human stool samples.


{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
12
{{< tdclose >}}
{{< tdopen >}}
Applications of NPs in Combating Diseases
{{< tdclose >}}
{{< tdopen >}}


{{< anchor "L12summary" >}}{{< /anchor >}}**Applying NPs to disease treatment involves testing effectiveness and mode of action**. Initial investigations of effectiveness can involve _in vitro_ cellular assays and _in vivo_ animal models of disease. Mode of action studies can be excessively challenging if no prior knowledge can guide the searchlight. This week we will discuss the application of a snail venom peptide for the treatment of liver cancer tumors and a PKS-NRPS hybrid peptide for the treatment of multi-drug resistant tuberculosis.

Anand et al. described their efforts to determine the mode of action of the snail venom peptide using fluorescence colocalization, protein expression analysis, and molecular modeling. Hartkoorn et al. focused on the pyridomycin mode of action by performing genomic sequencing of resistant tuberculosis strains in search of mutations in the target protein. They further determined the kinetics of inhibition and proposed important enzyme-inhibitor interactions. These studies highlight that pyridomycin binds to the same enzyme active site pocket as the clinically-used drug isoniazid.


{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
13
{{< tdclose >}}
{{< tdopen >}}
Student Presentations and Evaluation
{{< tdclose >}}
{{< tdopen >}}


{{< anchor "L13summary" >}}{{< /anchor >}}Students will give their oral presentations as outlined in the {{% resource_link 81d93051-8824-7c26-6d38-f900084d91e1 "Assignments" %}} section. Students also will complete a course evaluation.


{{< tdclose >}}

{{< trclose >}}

{{< tableclose >}}